docs/topics/auth/default.txt

```diff
@@ -231,6 +231,38 @@ to test for basic permissions you should use:
 The :class:`~django.contrib.auth.models.Permission` model is rarely accessed
 directly.
 
+Permission rename
+-----------------
+
+When a model is renamed in an installed application, Django will 
```

Comment:
Needs a `.. versionchanged::6.0` annotation when this is all fixed up. And a release note for 6.0.1.txt.